src/app/Component/chatroom/ChatRoom.js

```diff
@@ -114,17 +132,20 @@ class ChatRoom extends Component {
 
   imageUploading=(e)=>{
     console.log(e.target.files[0],'image event');
-      if (!e.target.files[0].name.match(/.(jpg|jpeg|png|gif)$/i))
-      {
-        alert('worng format of file');
-      }else{
-        if(e.target.files[0].size/1024<1024){
-          console.log('file is below 2mb and image format is also acceptable');
-        }
+    if (!e.target.files[0].name.match(/.(jpg|jpeg|png|gif)$/i))
+    {
+      alert('worng format of file');
+    }else{
+      if(e.target.files[0].size/1024<1024){
+        console.log('file is below 2mb and image format is also acceptable');
```

Comment:
Remove the console log

Reply:
api is not provided for image uploading.so we used console here